recipes/scipoptsuite/meta.yaml

```diff
@@ -0,0 +1,178 @@
+
+  - name: scip
+    version: {{ scip_version }}
+    build:
+      run_exports:
+        - {{ pin_subpackage('scip') }}
+    requirements:
+      build:
+        - {{ compiler('fortran') }}
+        - {{ compiler('c') }}
+        - {{ compiler('cxx') }}
+      host:
+        - tbb-devel
+        - libblas
+        - zlib
+        - gmp
+        - readline
+        - libgcc-ng  # [linux] Quadmath
+        - ipopt
+        - cppad
+    files:
+      - "lib/libscip*"
+      - "lib/cmake/scip/"
+      - "include/blockmemshell/"
+      - "include/dijkstra/"
+      - "include/lpi/"
+      - "include/nlpi/"
+      - "include/objscip/"
+      - "include/scip/"
+      - "include/symmetry/"
+      - "include/tclique/"
+      - "include/tinycthread/"
+      - "include/tpi/"
+      - "include/xml/"
+      - "bin/scip"
+    test:
+      script: run_scip_test.sh
+      source_files:
+        - scipoptsuite/scip/examples/Queens
+      requires:
+        - {{ compiler('cxx') }}
+        - cmake >=3.15
+        - make
+
+  - name: soplex
+    version: {{ soplex_version }}
+    requirements:
+      build:
+        - {{ compiler('cxx') }}
+      host:
+        - zlib
+        - gmp
+        - boost-cpp
+      run:
+        # libboost_program_options.so needed by bin/soplex.
+        # boost does not set run_exports so it is needed in run requirements.
+        - boost-cpp
+    files:
+      - "lib/libsoplex*"
+      - "lib/cmake/soplex/"
+      - "include/soplex*"
+      - "include/soplex/"
+      - "bin/soplex"
+    test:
+      script: run_soplex_test.sh
+      source_files:
+        - scipoptsuite/soplex/src/example.cpp
+      requires:
+        - {{ compiler('cxx') }}
+        - cmake >=3.15
+        - make
+
+  - name: gcg
+    version:  {{ gcg_version }}
+    build:
+      run_exports:
+        - {{ pin_subpackage('gcg') }}
+    requirements:
+      build:
+        - {{ compiler('c') }}
+        - {{ compiler('cxx') }}
+      host:
+        - gmp
+        - cliquer
+        - gsl
+        - {{ pin_subpackage('scip', exact=True) }}
+      run:
+        - {{ pin_subpackage('scip') }}
+        # run_exports not set for cliquer
+        - cliquer
+    files:
+      - "lib/libgcg*"
+      - "lib/cmake/gcg"
+      - "include/gcg"
+      - "bin/gcg"
+    test:
+      commands:
+        - gcg --version
+        - test -d "${PREFIX}/lib/cmake/gcg"
+        - test -d "${PREFIX}/include/gcg"
+
+  - name: zimpl
+    version: {{ zimpl_version }}
+    requirements:
+      build:
+        - {{ compiler('c') }}
+      host:
+        - zlib
+        - gmp
+        - bison
+        - flex
+    files:
+      - "lib/libzimpl*"
+      - "lib/cmake/zimpl/"
+      - "include/zimpl"
+      - "bin/zimpl"
+    test:
+      commands:
+        - zimpl -V
+        - test -d "${PREFIX}/lib/cmake/zimpl"
+        - test -d "${PREFIX}/include/zimpl"
+
+about:
+  home: https://scipopt.org/
+  license: ZIB-Academic
+  license_file:
+    - scipoptsuite/COPYING
+    - bliss/COPYING.LESSER
```

Comment:
This file doesn't have the copyright. Can you add that as well? https://github.com/mkoeppe/bliss/blob/sage_package/utils.hh#L5-L6